### PR TITLE
ootb/scanning: improve scantemplate instructions

### PR DESCRIPTION
**Pull request by cirocosta**
hi folks,

here in this PR I propose that  rather than having the template inlined in our documentation, we refer people to installation w/ `tanzu package install` instead and point at the documentation that the scst team maintains so that instead of having to keep that up to date in two places, we can link to the best source of truth for that.

wdyt @xtreme-conor-nosal @nedenwalker?

thank you!

---

marking as a draft so we can gather insights from other folks beforehand.

### CODE REVIEW

File: scc/ootb-supply-chain-testing-scanning.md [hunk 1]
```diff
@@ -163,118 +169,53 @@ spec:
...
+    ```
+    tanzu package install grype-scanner \
+      --package-name grype.scanning.apps.tanzu.vmware.com \
+      --version 1.0.0 \
+      --namespace my-dev-namespace
+    ```
+
+
+**note**: Although the templates can be customized, we recommend sticking with
+what's provided by the instalation of `grype.scanning.apps.tanzu.vmware.com`
```
Comment (xtreme-conor-nosal) [inline scc/ootb-supply-chain-testing-scanning.md:213]: typo: installation

File: scc/ootb-supply-chain-testing-scanning.md [hunk 2]
```diff
@@ -163,118 +169,53 @@ spec:
...
+
+    ```
+    tanzu package install grype-scanner \
+      --package-name grype.scanning.apps.tanzu.vmware.com \
+      --version 1.0.0 \
+      --namespace my-dev-namespace
+    ```
+
+
+**note**: Although the templates can be customized, we recommend sticking with
```
Comment (xtreme-conor-nosal) [inline scc/ootb-supply-chain-testing-scanning.md:212]: Might want to qualify the recommendation? For getting started it makes sense to suggest the OOTB scanner for the OOTB pipeline, but beyond that we don't want to discourage users from bringing their own scanner.

File: scc/ootb-supply-chain-testing-scanning.md [hunk 3]
```diff
@@ -163,118 +169,53 @@ spec:
...
+what's provided by the instalation of `grype.scanning.apps.tanzu.vmware.com`
+(created in the same namespace as configured via `grype.namespace` in either
+TAP profiles or individual component installation as in the example above). See
+[Creating a ScanTemplate](scst-scan/create-scan-template.md) for more details.
```
Comment (xtreme-conor-nosal) [inline scc/ootb-supply-chain-testing-scanning.md:216]: If they create a new scantemplate, they might also need to update the Scan CR `spec.scanTemplate` in the ClusterSourceTemplate/ClusterImageTemplate.

Comment (cirocosta) [reply scc/ootb-supply-chain-testing-scanning.md:216]: right now we're not instructing folks on how to modify the templates from ootb / supply chains we ship 🤔 let me rephrase it a bit

Comment (cirocosta) [reply scc/ootb-supply-chain-testing-scanning.md]: @xtreme-conor-nosal thx for the feedback - pushed two changes, wdyt?

Comment (xtreme-conor-nosal) [reply scc/ootb-supply-chain-testing-scanning.md]: LGTM

**Comment by RichardJJG** [comment]
Thanks, guys!
